openeo-geotrellis/src/main/scala/org/openeo/geotrellis/geotiff/package.scala

```diff
@@ -257,6 +257,14 @@ package object geotiff {
             // ':' is not valid in a Windows filename
             DateTimeFormatter.ISO_ZONED_DATE_TIME.format(key.time).replace(":", "").replace("-", "")
           }
+          val overviews = if(formatOptions.overviews.toUpperCase == "ALL" || (formatOptions.overviews.toUpperCase == "AUTO" && (gridBounds.width>1024 || gridBounds.height>1024 )) ) {
+            val decimationFactor = 4
```

Comment:
why 4?
Would `Pyramid.up` work here, like in the other pyramid calculation?  https://github.com/Open-EO/openeo-geotrellis-extensions/blob/aaf9914e8933d3e716a6332c98e3c7ffa0b0632d/openeo-geotrellis/src/main/scala/org/openeo/geotrellis/geotiff/package.scala#L551